site/README.md

```diff
@@ -1,25 +1,25 @@
-# Reamark42 Website
+# Remark42 site
```

Comment:
here is APA title case, it is used for first/second level of titles. I don't see error here